hack/verify-description.sh

```diff
@@ -37,13 +47,17 @@ find_files() {
     \) -name '*.go'
 }
 
-files=`find_files | egrep "pkg/api/v.[^/]*/types\.go"`
+if [[ "$#" -ne "0" ]]; then
+  files=$1
+else
+  files=`find_files | egrep "pkg/api/v.[^/]*/types\.go" | grep -v "v1beta3"`
+fi
 
 for file in $files
 do
-    if [[ "$("${KUBE_ROOT}/hooks/description.sh" "${file}")" -eq "0" ]]; then
-      echo "API file is missing the required field descriptions: ${file}"
-      result=1
+    $genswaggertypedocs -v -s "${file}" -f - || result=$?
```

Comment:
Shall it be 
```
$genswaggertypedocs -v -s "${file}" -f - && result=0 || result=$?
```
I think "result" will be unbound if $genswaggertypedocs runs successfully.